OpenRA.Game/Graphics/ChromeProvider.cs

```diff
@@ -154,12 +154,17 @@ public static Sprite GetImage(string collectionName, string imageName)
 
 			if (!collections.TryGetValue(collectionName, out var collection))
 			{
-				Log.Write("debug", "Could not find collection '{0}'", collectionName);
+				Log.Write("debug", $"Could not find collection '{collectionName}'");
 				return null;
 			}
 
 			if (!collection.Regions.TryGetValue(imageName, out var mi))
+			{
+				if (logNotFound)
+					Log.Write("debug", $"Could not find image '{imageName}' in collection '{collectionName}'");
```

Comment:
This results in several new lines in the log:
```
Could not find image 'corner-tl' in collection 'scrollitem'
Could not find image 'border-t' in collection 'scrollitem'
Could not find image 'corner-tr' in collection 'scrollitem'
Could not find image 'border-l' in collection 'scrollitem'
Could not find image 'background' in collection 'scrollitem'
Could not find image 'border-r' in collection 'scrollitem'
Could not find image 'corner-bl' in collection 'scrollitem'
Could not find image 'border-b' in collection 'scrollitem'
Could not find image 'corner-br' in collection 'scrollitem'
Could not find image 'corner-tl' in collection 'scrollitem'
Could not find image 'border-t' in collection 'scrollitem'
Could not find image 'corner-tr' in collection 'scrollitem'
Could not find image 'border-l' in collection 'scrollitem'
Could not find image 'background' in collection 'scrollitem'
Could not find image 'border-r' in collection 'scrollitem'
Could not find image 'corner-bl' in collection 'scrollitem'
Could not find image 'border-b' in collection 'scrollitem'
Could not find image 'corner-br' in collection 'scrollitem'
```

Reply:
This log should have been here from the very start. We're going to have to investigate this case by case

Reply:
fixed